### DISCUSSION TITLE
Possibilities for training on longer samples

### DISCUSSION THREAD

**Discussion by SashaBurashnikova** [cifkao/ss-vq-vae#4] [Q&A]
Hello,

Your results sounds very nice, thank you for sharing them in git!) i'm just wondering is it possible to apply your model to samples with longer duration? If yes, what is the relevant time-limit for samples in case of keeping the current output quality?

**Comment by cifkao** [comment]
Hi! If you mean _training_ on longer sequences, there shouldn't really be any limit as long as they fit in memory (you might just want to tune the batch size and learning rate). On the other hand, if you mean running the pre-trained model on longer samples, I would indeed expect the quality to degrade. But either way, I haven't tried this, so I cannot speak from experience.

**Reply by SashaBurashnikova** [reply]
Hello!

Thank you!) yes, i mean training and testing the model not only on 8s samples, but longer, like 1 minute duration)Its interesting will it possible to apply your model for the full music composition and provide timbre transfer on it)
I see, thank you very much for the answer!)
